tests/components/kostal_plenticore/test_number.py

```diff
@@ -136,7 +136,7 @@ async def test_set_value(
         mock_coordinator, "42", "scb", None, mock_number_description, mock_setting_data
     )
 
-    await entity.async_set_value(42)
+    await entity.async_set_native_value(42)
```

Comment:
@stegm this is not how we want to test entities, instead the entity should be added to hass' state machine, and the service should be called. Please address that in a follow-up 👍